todo_app/data/todoapi.py

```diff
@@ -0,0 +1,160 @@
+from abc import ABC, abstractmethod
+from configparser import ConfigParser
+from .boardelements import *
+import os
+import requests
+
+_TRELLO = 'trello'
+
+def todo_api_factory(api_name):
```

Comment:
This could become a `@classmethod` on the `TrelloAPI` class

Reply:
Obsolete - removed